test/e2e/predictor/test_lightgbm.py

```diff
@@ -54,7 +54,7 @@ def test_lightgbm_kserve():
     kserve_client.create(isvc)
     kserve_client.wait_isvc_ready(service_name, namespace=KSERVE_TEST_NAMESPACE)
 
-    res = predict(service_name, "./data/iris_input_v3.json")
+    res = predict(service_name, "./data/iris_input_v4.json")
```

Comment:
Can we test both inputs?

Reply:
There is already a test for another format